packages/validation-manager/src/TracerResultParser.ts

```diff
@@ -350,7 +350,9 @@ export function tracerResultParser (
         // slot associated with sender is allowed (e.g. token.balanceOf(sender)
         // but during initial UserOp (where there is an initCode), it is allowed only for staked entity
         if (associatedWith(slot, sender, entitySlots)) {
-          if (userOp.factory != null) {
+          console.log('WTF slot assoc with account', slot)
```

Comment:
Please do not include `console.log` in your commits

Reply:
Fixed